{
  "id": "CVE-2024-57722",
  "sourceIdentifier": "cve@mitre.org",
  "published": "2025-01-23T01:15:27.137",
  "lastModified": "2025-03-14T18:15:31.113",
  "vulnStatus": "Awaiting Analysis",
  "cveTags": [],
  "descriptions": [
    {
      "lang": "en",
      "value": "lunasvg v3.0.0 was discovered to contain a allocation-size-too-big bug via the component plutovg_surface_create."
    },
    {
      "lang": "es",
      "value": "Se descubri\u00f3 que lunasvg v3.0.0 conten\u00eda un error de tama\u00f1o de asignaci\u00f3n demasiado grande a trav\u00e9s del componente plutovg_surface_create."
    }
  ],
  "metrics": {
    "cvssMetricV31": [
      {
        "source": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
        "type": "Secondary",
        "cvssData": {
          "version": "3.1",
          "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
          "baseScore": 7.5,
          "baseSeverity": "HIGH",
          "attackVector": "NETWORK",
          "attackComplexity": "LOW",
          "privilegesRequired": "NONE",
          "userInteraction": "NONE",
          "scope": "UNCHANGED",
          "confidentialityImpact": "NONE",
          "integrityImpact": "NONE",
          "availabilityImpact": "HIGH"
        },
        "exploitabilityScore": 3.9,
        "impactScore": 3.6
      }
    ]
  },
  "weaknesses": [
    {
      "source": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
      "type": "Secondary",
      "description": [
        {
          "lang": "en",
          "value": "CWE-770"
        }
      ]
    }
  ],
  "references": [
    {
      "url": "https://github.com/keepinggg/poc/blob/main/poc_of_lunasvg_3.1.0",
      "source": "cve@mitre.org"
    },
    {
      "url": "https://github.com/sammycage/lunasvg/issues/209",
      "source": "cve@mitre.org"
    }
  ]
}